### PR TITLE
feat: add feat config and local file storage

### PR DESCRIPTION
**Pull request by Yoshino-s**

### CODE REVIEW

**Comment by Yoshino-s** [comment]
related issue #959 #344

**Comment by boojack** [comment]
And I'd like to split this PR into 2 pieces:

1. introducing "github.com/spf13/cobra"&"github.com/spf13/viper" and the feature flags;
2. implement local file storage.

WDYT

**Comment by Yoshino-s** [comment]
> And I'd like to split this PR into 2 pieces:
> 
> 1. introducing "github.com/spf13/cobra"&"github.com/spf13/viper" and the feature flags;
> 2. implement local file storage.
> 
> WDYT

LGTM，i will revert last commit

File: main.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	_ "github.com/mattn/go-sqlite3"
+
+	"github.com/usememos/memos/cmd"
+)
+
+func main() {
```
[the remark this answers is not in the document]
Comment (Yoshino-s) [reply main.go:9]: the cobra said another practice https://github.com/spf13/cobra/blob/main/user_guide.md

**Pull request closed without merging**